src/sentry/utils/tag_normalization.py

```diff
@@ -0,0 +1,92 @@
+import functools
+import re
+
+_KNOWN_TAGS = {
```

Comment:
@ellisonmarks the unique values for SDK tag would be reduced to this list + _"other"_, so as it stands now, 42 unique values. Is that an acceptable cardinality?

Reply:
@vartec - after our iterations, what's the new count?

Reply:
@lobsterkatie 36 total (35 known + "other")

Reply:
Given that we started with, what, 270ish? that’s really not too shabby! 🙂